cx_Freeze/windist.py

```diff
@@ -44,6 +44,9 @@ def add_config(self, fullname):
             msilib.add_data(self.db, "Directory", self.directories)
         if self.environment_variables:
             msilib.add_data(self.db, "Environment", self.environment_variables)
+        # This is needed in case the AlwaysInstallElevated policy is set.
+        # Otherwise installation will not end up in TARGETDIR.
+        msilib.add_data(self.db, "Property", [("SecureCustomProperties", "TARGETDIR")])
```

Comment:
What happens if the AlwaysInstallElevated policy is *NOT* set? Does this cause a problem in that case? If not, I'd be happy to merge this. I don't use Windows regularly enough to know the answer or easily create a test.